modules/account-lib/src/index.ts

```diff
@@ -62,6 +62,9 @@ export { Stx };
 import { AlgoLib as Algo } from '@bitgo/sdk-coin-algo';
 export { Algo };
 
+import * as Arbeth from '@bitgo/sdk-coin-arbeth';
```

Comment:
Unless there is a specific use case for using this in account-lib elsewhere, let's remove it.

Reply:
We are using this getBuilder [method](https://github.com/BitGo/BitGoJS/pull/3971/files#diff-9189ca44944c5c699c945493c6c19b77b239dab907b726f164eb76fcd976a8a4R181) in wallet platform, so we will have to add this coin here